core/src/main/java/bio/terra/pearl/core/service/notification/email/AdminEmailService.java

```diff
@@ -80,6 +84,8 @@ public boolean shouldSendEmail(NotificationContextInfo contextInfo) {
    */
   public NotificationContextInfo loadContextInfo(String templateStableId, int version, Portal portal) {
     EmailTemplate emailTemplate = emailTemplateService.findByStableId(templateStableId, version).get();
+    //There are no current plans to localize the admin tool, so we just attach the English version
```

Comment:
let's attach all the templates here -- we send out admin emails so rarely it's not worth optimizing.  And then we only have to hardcode "en" in one place